debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -225,6 +225,18 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
             .withValidation(Field::isOptional)
             .withDescription("The SQL Server instance name");
 
+    public static final Field DATABASE_NAME = RelationalDatabaseConnectorConfig.DATABASE_NAME
+            .withNoValidation()
+            .withValidation(SqlServerConnectorConfig::validateDatabaseName);
```

Comment:
I'll remove this one, so to avoid a validation error on _both_ fields in case both are given. Displaying it once at the new property should suffice.